src/map/utils/mobutils.cpp

```diff
@@ -54,19 +54,33 @@ namespace mobutils
 
     uint16 GetWeaponDamage(CMobEntity* PMob, uint16 slot)
     {
-        uint16 lvl   = PMob->GetMLevel();
-        uint8  bonus = 0;
+        uint16 lvl    = PMob->GetMLevel();
+        int8   bonus  = 0;
+        uint16 damage = 0;
 
-        if (slot == SLOT_RANGED)
+        if (lvl >= 75)
```

Comment:
I would take this block and re-nest things so that it checks for the ranged, and puts this cascading if inside that else block.
```
if (slot == SLOT_RANGED)
{
   bonus = 5;
}
else
{
    if (lvl >= 75)
    {
        bonus = 3;
     }
    else if (lvl >= 60)
    {
        bonus = 2;
    }  // ... etc
} // Closes the initial if

damage = lvl + bonus;
```